components/HomeScreenComp/CompeteAndWin/CompeteAndWin1.js

```diff
@@ -37,23 +39,23 @@ const CompeteAndWin1 = () => {
           <CompeteAndWinCard
```

Comment:
Mridul , you have to loop around the card.  you should not duplicate the code for every card. use .map function or similar.